packages/UI/src/docs/System/Iconography.mdx

```diff
@@ -13,6 +12,17 @@ import * as Icons from '../../assets/icons';
       <IconJSX />
     </IconItem>
   ))}
+
+</IconGallery>
+
+## Lucide Icons
+
+<IconGallery>
+  {Object.entries(icons).map(([IconName, IconJSX]) => (
+    <IconItem name={IconName} key={IconName}>
+      <IconJSX />
+    </IconItem>
+  ))}
 </IconGallery>
 
```

Comment:
This is a good start but to make it easier to navigate maybe we should put lucide react icons in a collapsible section with guide on how to import them.

same thing to be said about custom-made icons.

something like this

https://components-iconography-suggestion--632ca21aeea1618b7fa0bf48.chromatic.com/?path=/docs/iconography--docs

code that made it happen https://github.com/dev-launchers/dev-launchers-platform/pull/1688